src/hermes/commands/harvest/cff.py

```diff
@@ -74,8 +74,19 @@ def _load_cff_from_file(cff_data: str) -> t.Any:
     return yaml.load(cff_data)
 
 
+def _patch_author_emails(cff_data, codemeta_str):
+    cff_authors = _load_cff_from_file(cff_data)["authors"]
```

Comment:
**nitpick**: I wonder if it would be more concise when not loading the data again by moving the call to this method from `_convert` up one level, behind calling `_convert` (line 66) and pass `cff_dict`.